### ISSUE TITLE
[Docs] Bitbucket Pipelines documents /buildserver CLI option

### ISSUE THREAD

**Issue by dhilgarth** [GitTools/GitVersion#3498]
https://gitversion.net/docs/reference/build-servers/bitbucket-pipelines

The documentation for Bitbucket pipelines states to use `dotnet-gitversion /buildserver` but this CLI option doesn't exist.  
It should be `dotnet-gitversion /output buildserver` instead.

<details>

```
dotnet-gitversion /buildserver
Unhandled exception. GitVersion.WarningException: Could not parse command line parameter '/buildserver'. If it is the target path, make sure it exists.
   at GitVersion.ArgumentParser.ParseTargetPath(Arguments arguments, String name, IReadOnlyList`1 values, String value, Boolean parseEnded) in D:\a\GitVersion\GitVersion\src\GitVersion.App\ArgumentParser.cs:line 190
```

![image](https://user-images.githubusercontent.com/1180105/235851242-34abcc8a-85bf-4cf5-a554-78d8e6d90959.png)
</details>

**Comment by arturcic** [comment]
@dhilgarth do you mind sending us a PR with the changes?

**Comment by dhilgarth** [comment]
Sure, done

**Comment by asbjornu** [comment]
Resolved in #3499.
